Documentation/aws.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Running Rook in AWS
+weight: 25
+indent: true
+---
+
+# Running Rook in AWS
```

Comment:
This seems like another version of the blog, but without all of the details. It feels like we either should put this in the design docs folder, or turn this into a more detailed walkthrough that shows the steps of how to setup Rook in AWS, or just rely on the blog and not merge this.

Reply:
Yeah i kinda feel the same way.

Reply:
I think there's a benefit in having a documentation section for some provider specific Rook considerations, for AWS, GCE,  etc. They wouldn't necessarily describe how to install k8s, there are plenty of projects for that, or how to install Rook, since the installation is no different.

If that is not needed, this request should be closed.